judge/forms.py

```diff
@@ -230,6 +232,22 @@ def clean(self):
             raise ValidationError(_('Must specify either totp_token or webauthn_response.'))
 
 
+class WCIPEGMergeRequestForm(Form):
+    handle = CharField(max_length=50, validators=[UnicodeUsernameValidator()])
+
+    def clean_handle(self):
+        username = self.cleaned_data['handle']
+        if not User.objects.filter(username=username, is_active=True).exists():
+            raise ValidationError(_("Account doesn't exist."))
+        user = User.objects.get(username=username)
+        if Profile.objects.get(user=user).is_peg():
```

Comment:
Stiil don't think this is correct. `is_peg` is a property, and you'll likely get an error similar to
```py
TypeError: 'bool' object is not callable
```

Reply:
Right... back to what it was before.